spec/index.html

```diff
@@ -324,39 +325,46 @@ <h3>Shared blank nodes, unions and merges</h3>
       to identify a blank node, but unless these documents are in a shared identifier scope
       or are derived from a common source,
       the occurrences of "<code>_:x</code>" in one document
-      will identify a different blank node than the one in the graph described by the other document.
+      will identify a different blank node than the occurrences of "<code>_:x</code>"
+      in the graph described by the other document.
       When graphs are formed by combining RDF from multiple sources,
       it may be necessary to <dfn class="no-export lint-ignore">standardize</dfn> apart the blank node identifiers
-      by replacing them by others which do not occur in the other document(s).
-      For example, the two graphs represented  by the following texts:</p>
+      by replacing them with other identifiers which do not occur in the other document(s).
+      For example, the two graphs represented by the two texts below contain two nodes each, 
+      for a total of four nodes:</p>
 
     <p><code>ex:a ex:p _:x . </code><br/><br/>
       <img src="RDF12SemanticsDiagrams/example1.svg" alt="Graph 1" /></p>
 
     <p><code>ex:b ex:q _:x . </code><br/><br/>
       <img src="RDF12SemanticsDiagrams/example2.svg" alt="Graph 2" ></p>
 
-    <p>contain four nodes. Their union would therefore also contain four nodes:</p>
+    <p>Their union would also contain four nodes:</p>
 
     <p><img src="RDF12SemanticsDiagrams/example4.svg" alt="Union Graph"></p>
 
-    <p>However, the document formed by simply concatenating these textual surface representations:</p>
+    <p>However, if we simply concatenate these textual surface representations 
+      to form a new document, as shown below:</p>
 
     <p><code>ex:a ex:p _:x .<br/>
     ex:b ex:q _:x .</code><br/></p>
 
-    <p>describes a graph containing three nodes:</p>
+    <p>This new document describes a graph containing three nodes, because
+      the two occurrences of the blank node identifier "<code>_:x</code>
+      now occur in a common identifier scope, and thus identify the same blank node,
+      as shown below:</p>
 
     <p><img src="RDF12SemanticsDiagrams/example3.svg" alt="Incorrect Union Graph"></p>
 
-    <p> since the two occurrences of the blank node identifier "<code>_:x</code>" occurring in a common identifier scope identify the same blank node. The four-node union of these two graphs is more properly described by a surface form such as:</p>
+    <p>The four-node union
+      of these two graphs is more properly described by a surface form such as the following:</p>
```

Comment:
```suggestion
      of these two graphs is instead:</p>
```

Reply:
You can only have the colon if it is preceded by a complete sentence, like the following:
```suggestion
      of these two graphs is instead that shown below:</p>
```